docs/conflicts.md

```diff
@@ -184,3 +184,32 @@ New Heading
 ===========
 >>>>>>>>>>>>>>> Conflict 1 of 1 ends
 ```
+
+## Conflicts with missing terminating newline (`[noeol]`)
+
+On POSIX systems, text files are generally assumed to be a series of lines with
```

Comment:
Suggestion: (feel free to reword it further or not to use it)

> `jj` relies on line-based diff algorithms, which work best on text files that consist of a series of lines with each line terminated by a newline character. This means that a text file would either be empty or it would terminate in a newline.
>
> While most text files follow this convention, some do not. When `jj`....

Reply:
Thanks, LGTM.